exercises/word-search/HINTS.md

```diff
@@ -0,0 +1,3 @@
+## Hints
+Tuples can be used to provide access to and manipulation of a data set.  For example, you can use them to pass multiple values to a method through a single parameter.
```

Comment:
Although I like the phrasing, it sort of implies that you *could* use tuples, whereas in fact you *have* to use tuples. Also, it doesn't list on of the prime use cases of tuples: returning multiple values from a function. Coincidentally, it is for returning multiple values from a function that this exercise uses tuples.

So perhaps you could rephrase this a bit to note that this exercise requires you to write a function that returns a tuple, which we use to allow us to easily return two values: the x- and y part of a coordinate.